cms/static/js/certificates/views/certificate_preview.js

```diff
@@ -0,0 +1,41 @@
+define([
```

Comment:
Can we add a descriptive comment at the top of this file describing how the preview feature works (used for cycling through course modes, opens new tab in LMS, etc.)